packages/core/src/components/menu/_submenu.scss

```diff
@@ -16,7 +16,7 @@
     &.#{$ns}-popover-open > .#{$ns}-menu-item {
       // keep a trail of hovered items as submenus are opened
       /* stylelint-disable-next-line scss/at-extend-no-missing-placeholder */
-      @extend .#{$ns}-menu-item:hover;
+      @extend .#{$ns}-menu-item, :hover;
```

Comment:
Have you tested this end-to-end? [More details here](https://github.com/palantir/blueprint/issues/4032#issuecomment-1039940037)

Reply:
Thanks for finding this.
We can't just not make this change, as compound selector extends are not valid sass in dart-sass.

I've used a placeholder value, as suggested [here](https://sass-lang.com/documentation/breaking-changes/extend-compound) in the "Heads Up!" section to get around the issue, but that might have it's own issues.